chart/values.yaml

```diff
@@ -152,6 +347,18 @@ kube-prometheus-stack:
       adminUser: postgres
       adminPassSecret: *dbPassSecret
 
+  # By default kube-state-metrics are scraped using
+  # serviceMonitor disable annotation based scraping
+  kube-state-metrics:
+    prometheusScrape: false
+
+  # By default node-exporter are scraped using
+  # serviceMonitor disable annotation based scraping
+  prometheus-node-exporter:
+    service:
+      annotations:
+        prometheus.io/scrape: "false"
```

Comment:
do we want this in the `cli/tests/testdata/main-values.yaml ` file too?

Reply:
Not really. As in our test we do not care about targets and the metrics that are being scraped. So all the additional config only exists in values.yaml that are shipped with default tobs helm chart.